locales/en/app.ftl

```diff
@@ -562,3 +562,97 @@ rec-section-subhead-no-pw = Though passwords weren’t exposed in this breach, t
+confirmation-2-subhead = Take that, hackers!
+confirmation-2-body = You’re taking important steps towards protecting your online accounts.
+confirmation-3-subhead = Another one down. Nice work!
+
+# This string contains nested markup that becomes a link later in the code.
+# Please do not modify or remove "<a>" and "</a>".
+confirmation-3-body = Is your new password unique, strong, and hard to guess? <a>Find out</a>
+
+generic-confirmation-1-subhead = This breach has been marked as resolved
+generic-confirmation-status =
+  { $numRemainingBreaches ->
+      [one] There is { $numRemainingBreaches } unresolved breach remaining.
+     *[other] There are { $numRemainingBreaches } unresolved breaches remaining. 
+  }
+generic-confirmation-complete = You’ve marked all known breaches as resolved.
+
+# Link
+return-to-breach-details = Return to breach details
+
+# Link
+go-to-dashboard = Go to Dashboard
+
+# This string appears above a breach resolution progress bar and indicates
+# the percentage of breaches a user has resolved. For instance, "27% complete".
+progress-percent-complete = { $percentComplete } complete
+
+# This string appears in the purple callouts at the top of the user dashboard and shows
+# the total number of breaches a user has resolved. For instance, "5 Resolved".
+num-resolved = { $numResolved } Resolved
+
+progress-intro-subhead = New in { -product-name }: Mark breaches as resolved
+progress-intro-message =
+  After reviewing the details about a breach and taking steps to protect 
+  your personal info, you can mark breaches as resolved.
+progress-status = { $numResolvedBreaches} out of { $numTotalBreaches } breaches marked as resolved
+progress-complete = All known breaches have been marked as resolved
+
+## These strings contain nested markup that is later used to style the text inside of it.
+## Please do not modify or remove "<span>" and "</span>".
+progress-message-1 = 
+  <span>You’re off to a great start!</span> Check out the remaining breaches to learn 
+  what steps to take.  
+progress-message-2 = 
+  <span>Keep it up!</span> Small changes like updating passwords have a big impact on 
+  keeping your personal info safe.
+progress-message-3 = <span>Nice work resolving those breaches!</span> Keep it up. You’ve got a few more to go.
+progress-message-4 = <span>Nice work resolving those breaches!<span> Keep it up. You’ve got a few more to go.
+progress-message-5 = <span>Almost done!</span> You’re nearly to the finish line.
+progress-complete-message = 
+  <span>Feels good, right?</span> If you want to keep going, this is a good time to 
+  update other logins with stronger passwords.
+##
+
+# A status indicator that appears in the top right corner of breach cards
+# once they have been resolved.
+resolved = Resolved
+
+# Link
+resolve-this-breach = Resolve this breach
+
+# This string appears in resolved breach cards and is followed by 
+# the date the user marked the breach as resolved.
+marked-resolved = Marked resolved:
```

Comment:
How is the date going to be displayed? Can it be exposed as part of the string?

Reply:
Here is how it will show up in resolved breach cards:
<img width="475" alt="Screen Shot 2019-12-18 at 4 54 16 PM" src="https://user-images.githubusercontent.com/22355127/71130352-40e98900-21b7-11ea-856d-ce6a6758c72b.png">
We could expose the date in the string but we'd probably have to add markup around the date and then do some additional juggling in the template.